.github/workflows/python-app.yml

```diff
@@ -27,6 +27,9 @@ jobs:
         python -m pip install --upgrade pip
         pip install flake8 pytest
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f requirements-dev.txt ]; then pip install -r requirements-dev.txt; fi
+        pip install -e .
+        mkdir /tmp/testing
```

Comment:
Unless files are inspected following testing we can use the [`tempfile` lilbrary](https://docs.python.org/3/library/tempfile.html) to prevent file system cluttering.

```py
with tempfile.TemporaryDirectory() as dest:
    runner = CliRunner()
    result = runner.invoke(cli, ['terra', '-d', dest])
    assert result.exit_code == 0

    files = sorted(next(os.walk(dest))[2])
    assert len(files) == 10
    assert files[0] == "HG00536.final.cram.crai"
    assert files[9] == "NA20525.final.cram.crai"
```